### PR TITLE
Add ubuntu-bionic-i386

### PR DESCRIPTION
**Pull request by dstebila**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -23,6 +23,9 @@ localCheckout: &localCheckout
     #- run:
     #    name: Ubuntu-Bionic
     #    command: cd ubuntu-bionic && docker build --build-arg ARCH=x86_64 -t ci-ubuntu-bionic . && docker tag ci-ubuntu-bionic $TARGETNAME/ci-ubuntu-bionic-x86_64
+    - run:
+        name: Ubuntu-Bionic-i386
+        command: cd ubuntu-bionic && docker build --build-arg ARCH=i386 -t ci-ubuntu-bionic . && docker tag ci-ubuntu-bionic $TARGETNAME/ci-ubuntu-bionic-i386
```
Comment (baentsch) [inline .circleci/config.yml:28]: Why bionic (18)? Does LTS focal (20) cause problems with i386?

**Review by baentsch — CHANGES_REQUESTED** [review]
If i386-focal causes problems, please let me know: I'd look into it in order to keep everything at Ubuntu 20.

File: .circleci/config.yml (hunk 1 above)
Comment (dstebila) [reply .circleci/config.yml:28]: Ubuntu stopped supporting 32-bit with Focal; Bionic is the last 32-bit Ubuntu LTS.

Comment (baentsch) [reply .circleci/config.yml:28]: OK - didn't know that. Thanks.

**Review by baentsch — APPROVED** [review]

**Pull request merged**